docs/platform/porting/system.md

```diff
@@ -2157,3 +2157,384 @@ $ sensorctl test accelerometer 100 1000 /* enable accelerometer with interval 10
+
+-   **stacks** (list): Contains settings for what should be dumped.
+    -   **dump_stacks** (bool): If true, stacks will be dumped.
+    -   **first_thread_only** (bool): If true, only the first thread will be dumped.
+    -   **max_stack_size** (integer): Maximum size of the stack.
+-   **maps** (list): Contains list "dump_by_name".
+    -   **dump_by_name** (list): List of pathnames of memory regions that should be dumped (pathnames are read from `/proc/<PID>/maps`).
+-   **compression** (list): Contains settings related to report compression.
+    -   **compressor** (string): Command line of compressor that supports stdin as input and stdout as output (gzip, bzip2, xz).
+    -   **extension**: (string): In case of a compressed core file, the specified value will be appended at the end of the core file name; otherwise, `.compressed` will be appended to the compressed core files.
+    -   **in_tar** (bool): Determines whether the core file should be a `.tar` file or not. This is useful because the tar format enables preserving the sparse properties of the core file.
+    -   **dump_auxv_so_list** (bool): If true, the shared object list will be saved in the core file.
+    -   **dump_pthread_list** (bool): If true, the pthread list will be saved in the core file.
+    -   **dump_robust_mutex_list** (bool): If true then robust mutexes will be saved in the core file.
+    -   **dump_scope** (int): Only registered dumps at this value or lower than this value will be dumped. This option applies only to libminicoredumper.
+    -   **dump_build_id** (bool): If true, then for all the contained ELF files, minicoredumper will save ELF header with BuildID to enable determining a specific version of the shared libraries.
+    -   **live_dumper** (bool): If true, then minicoredumper will trigger applications registered by libminicoredumper.
+    -   **write_proc_info** (boolean): If true, then interesting files (`cmdline`, `environ`, `io`, `maps`, `smaps`, `stack`, `stat`, `statm`, `cwd`, `fd`) from `/proc/<PID>` will be copied to the dump directory.
+    -   **proc_info_exclude** (list): Contains a list of filenames that must not be copied from the `/proc/` directory. For example, copying the `smaps` files will significantly increase the dumping time.
+    -   **write_debug_log** (bool): If true, then minicoredumper messages will be saved to `debug.txt` in the dump directory.
+    -   **dump_fat_core** (bool): If true, then all virtual memory areas will be saved in the separate core dump file. This must be used only for debug purpose.
+    -   **dump_pointed_by_regs** (bool): If true, then minicoredumper saves in the core file memory region around the address to which the CPU  registers point to. Precisely, for each CPU register minicoredumper saves page to which address points, one page before the address, and one page after the address. Page is 4096 bytes on most architectures.
```

Comment:
Suggest rephrasing the first line to:

minicoredumper saves the core file memory region around the address to which the CPU registers point.